pipelines/fetch-farcaster-data/fetch.py

```
@@ -0,0 +1,191 @@
+            headers = {
+                'accept': 'application/json',
+                'api_key': self.NEYNAR_API_KEY
+            }
+            params = {
+                'fid': fid, 
+                'limit': 100,
+            }
+            try:
+                additional_channels = helpers.query_neynar_api(endpoint, params, headers)
+                return additional_channels
+            except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)